src/Compilers/VisualBasic/Portable/VisualBasicParseOptions.vb

```diff
@@ -29,7 +29,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' <param name="kind">The kind of source code.<see cref="SourceCodeKind"/></param>
         ''' <param name="preprocessorSymbols">An enumerable sequence of KeyValuePair representing preprocessor symbols.</param>
         Public Sub New(
-            Optional languageVersion As LanguageVersion = LanguageVersion.VisualBasic14,
+            Optional languageVersion As LanguageVersion = LanguageVersion.VBnext,
```

Comment:
On the C# side it looks like we're leaving this to default to the last shipped version. Changing this default means updating test cases that assume this version of the parser. We might want to push that off until we change the default to VisaulBasic15 later. That means your tests will have to manually specify a different version but all of the changes are isolated to new code you introduce.